projects/lib-shared-modules/src/lib/components/comments-box/comments-box.component.ts

```diff
@@ -28,7 +28,7 @@ export class CommentsBoxComponent implements OnInit {
   @Input() messages:any;
   @Output() comment = new EventEmitter<String>();
   value: any;
-  chatFlag: boolean = false;
+  chatFlag: boolean = true;
```

Comment:
why is this true ?

Reply:
https://katha.shikshalokam.org/bug-view-1718.html
according to this issue , comment box should open when we come to page 
so made it true